de.tudarmstadt.ukp.wikipedia.api/src/test/java/de/tudarmstadt/ukp/wikipedia/api/PageTest.java

```diff
@@ -105,15 +108,18 @@ public void testGetText() {
 	}
 
 	@Test
-	@Ignore // FIXME see #161 and #160
 	public void testGetPlainText() {
-		String expectedPlainText = "Wikipedia API ist die wichtigste Software überhaupt. Wikipedia API.\n" +
-				"Nicht zu übertreffen.\nUnglaublich\nhttp://www.ukp.tu-darmstadt.de\nen:Wikipedia API fi:WikipediaAPI";
+		String expectedPlainText = "Wikipedia API ist die wichtigste Software überhaupt.\n" +
+				"Wikipedia API.\n" +
+				"Nicht zu übertreffen.\n" +
+				"Unglaublich\n" +
+				"http://www.ukp.tu-darmstadt.de\n" +
+				"en:Wikipedia API fi:WikipediaAPI";
 		try {
```

Comment:
Instead of catching the exception, just declare it on the test method. JUnit does the right thing.

Reply:
Well, than I would have to change all other tests this way..., yet: this is not the topic of this PR.

Reply:
@mawiesne I agree with you, we should probably create a parent issue for removing the try catch pattern in all the test cases. However, since your PR touches this test case specifically, I wonder like @reckart whether it would be better just to remove the pattern here already.

Reply:
@tgalery Thanks for commenting. Yet, I do not agree to mix up different aspects in one PR. This should be discussed and changed in a separate issue/PR. Why: People introduce different code styles depending on their personal background. Nobody will ever be happy with every style that is in the code base already. However, one can find consensus on it when focussing only on the aspect of how Tests should be written.

In this PR, and I repeat myself, the topic is: "Fix a critical bug" not "beautifiy test code".